packages/codemods/README.md

```diff
@@ -0,0 +1,83 @@
+# ts-morphin-migration
+
+ts-morphin-migration is a package created and managed with the `create-just` utility.
+
+## Next Steps
+
+Run
+
+```sh
+yarn
+yarn start-test
+```
+
+To start testing the codemods
+
+## To add a codemod
+
+Add your codemods to the `./src/mods` folder with `.mod.ts|tsx` as the file type.
+
+## Test with the actual package:
+
+Run
+
+```sh
+yarn build
+```
+
+To build the upgrade.js file
+
+Run
+
+```sh
+npm pack
+```
+
+from the codemods root to create a tar file for testing. Move the created tar file to the package you want to test and run
+
+```sh
+npx <tarFileName>
+```
+
+## Executing a codemod (Prototype)
+
+Make sure that fluent is installed in your package.
+Run
+
+```
+npx @fluentui\codemods
```

Comment:
just want to confirm these will be supported:
1. User can pick a specific codemod to run, e.g: repath to compat path
2. User can target a specific component in case of repathing

Reply:
I don't believe that the goal is to support this for V1. We can talk more about it, however.

Reply:
I left this as an Ask in hackmd. we can discuss further with the team. regardless, this is something outside of the scope of this PR